src/cosalib/build.py

```diff
@@ -248,9 +272,18 @@ def __get_json(self, name):
         :type name: str
         :returns: dict
         """
+
         file_path = self.__file(name)
         log.debug("Reading in %s", file_path)
-        return load_json(file_path)
+        try:
+            return load_json(file_path)
+        except FileNotFoundError:
+            e = self._exceptions.get(name)
+            if e:
+                raise e(f"{file_path} is required")
+            else:
+                log.warn(f"{file_path} was not found, but is not required")
```

Comment:
Seems like a normal case, not something to warn about.

Reply:
Removed.